governance/SDK.md

```diff
@@ -0,0 +1,163 @@
+
+The voting criteria are:
+
+- 1 week to vote
+- 2/3 of the agreement to accept the vote
+- All _sdk maintainers_ are entitled to vote
+
+### Archive a project
+
+If a project is not following the criteria 1, 2, 4, 5, 6, 7, 8, the community MAY decides to archive the project. Prior to archiving, the community SHOULD first consider performing a search for a new maintainer to [handover the project](#handover-to-a-new-maintainergroup-of-maintainers).
+
+If no new maintainer is found, the community can proceed to archive the project
+using the [Asynchronous voting process](#asynchronous-voting-process).
+
+The voting criteria are:
+
+- 2 weeks to vote
+- 2/3 of the agreement to accept the vote
+- All _sdk maintainers_ are entitled to vote
+
+## Asynchronous voting process
+
+In order to ensure the democratic management of debates in the community, we
+define an asynchronous voting process using GitHub issues.
+
+The voting is defined by:
+
+- The proposed community change to vote
+- The set of people/groups entitled to vote
+- A voting period
+- The percentage of agreements required to accept the change
+- A voting manager, a member of the community entitled to manage the vote. The voting manager MUST be part of people entitled to vote.
+
+To start a vote, the voting manager opens an issue in the
+[CloudEvents Spec](https://github.com/cloudevents/spec/issues) with:
+
+- A binary question to be voted on
+- Tags of the entitled people/groups to vote (depending on the question)
+- The expiration date of the vote
+- The percentage of agreements to accept the change
+
+For example:
+
+> Question: Do you want to add "Paolo Rossi" @paolorossi as _sdk-go-maintainer_?
+>
+> Entitled to vote:
+>
+> - @sdk-go-maintainers
+>
+> Voting ends: 24th June 2020 12:00 CEST
+>
+> Required agreement: 50% + 1
+
+Then every person entitled can vote, up to the expiration date, commenting on
+the issue with a `+1` to express agreement or `-1` to express disagreement.
+Every comment except `+1` and `-1` won't be taken in account, because we expect
+that the voting process comes only after a debate on the change, developed on
+the community channels and Serverless WG/CloudEvents SDK meetings. In order to
+ensure visibility to the voting, people should be remembered to vote during
+Serverless WG meetings and community channels (Slack, ...).
+
+At the end of the voting period, the voting manager will count the votes,
+writing on the issue the results. Votes after the expiration date MUST not be
+considered; To calculate the agreement percentage, the voting manager should
+account the people who have expressed a vote, without taking in account the
+abstentions.
+
+If the vote passes, the voting manager should proceed with the change. If the vote
+does not pass, then a new vote for the same change MUST NOT happen for the
+next 6 months.
+
+This voting process MUST be used only to resolve the issues identified in this
```

Comment:
I'm not sure if this was the intent, but are you saying that this voting process can NOT be used for other topics? I'm wondering if you meant to say:
This voting process must be used in the following cases, but can be used in others as well:

Reply:
My intent is to avoid people starting _abusing_ of this (heavy) process for things that doesn't really need it.
If we want to add in future use cases where we want to use this process, then it's fine, we just need to change this document and agree on the changes